guidelines/05-javadoc-submission-process.markdown

```diff
@@ -0,0 +1,116 @@
+# Javadoc Submission Process
+
+The production of Javadoc with Liferay code is highly recommended and essential
+for anyone developing for Liferay products. The submission processes outlined
+in this article must be followed to ensure quality and efficiency during the
+review process. Follow the appropriate process matching your relationship with
+Liferay, Inc.
+
+## Liferay Internal Developers
+
+The creation of Javadoc is the responsibility of the developers who
+write/maintain its related code. This ensures that the documentation is correct
+and useful. There are two paths you can take for providing Javadoc for your area
+of expertise:
+
+- [Write Javadoc during the code creation/modification process.](#write-javadoc-during-code-creation)
+- [Write Javadoc for completed code that's already merged and available.](#write-javadoc-for-mergedavailable-code)
+
+Follow the path corresponding to your current workflow.
+
+### Write Javadoc During Code Creation
+
+Developers are allowed to submit Javadoc **with** code if certain precautions
+are followed (outlined below). Follow the below process for submitting Javadoc
+this way:
+
+1.  Write Javadoc for code areas that you have deep knowledge for. Be sure to
+    follow the
+    [Javadoc Guidelines](https://dev.liferay.com/participate/javadoc-guidelines)
+    when writing Javadoc to ensure the quickest review times.
```

Comment:
I noticed this argument for following the guidelines. Here are reasons why I think devs should follow the Javadoc Guidelines are because they ...
- help with wording
- ensure comprehensive descriptions and consistent style

Quick review times are nice, but reasons like the ones I mention convince me that the guidelines help me and the reader.